packages/@okta/vuepress-site/guides/sign-into-spa/section10/angular/getuserinfo.md

```diff
@@ -1 +1,19 @@
-placeholder
\ No newline at end of file
+User info ("claims") are available using the `getUser` method on the OktaAuthService.
```

Comment:
User info ("claims") are available using the `getUser` method on the `OktaAuthService`.